generate-model.yml

```diff
@@ -0,0 +1,15 @@
+version: '3'
```

Comment:
Having a file that is not documented seems bad form, either document how to use it or move it to another PR.